README.md

```diff
@@ -433,6 +433,50 @@ const cypressConfig = defineConfig({
 export default cypressConfig;
 ```
 
+#### Plugin Configuration
+
+A fifth argument is used for further configuration of the plugin.
+
+##### protectProduction
+
+The plugin tries to detect whether or not the firebase emulators are running, based on the respective environment variables being set or not. When the an emulator isn't running, production could be targeted instead which might be dangerous. The `protectProduction` key configures the behaviour when the emulator for a specific firebase service hasn't been detected. The options for the behaviour when an emalator is not running are:
+
+- `'none'` _(default)_ no protection is granted in this case, nor will a warning be output to the console. Production could be targeted.
```

Comment:
Do you think there is value in offering this option at all? Since not providing a setting here is the same as default

Reply:
Good point. I might have added it for (explicit) completeness of the different behaviours.
I do think it can be useful in case you have the protection turned off for a certain service and turned on for others.
In that case it has some 'warn' and/or 'error' entries. Others or forgetful future selves might quickly assume there is some protection for every service, when there might not be for a service that isn't listed. So being able to set it to 'none' explicitly shows the intent or the possible danger that is indeed turned off.
Of course it doesn't prevent users from not explicitly marking the protection is turned off, but at least you can do so if you would want to, explaining with a comment why it is turned off.